packages/manageiq-ansible-venv/Dockerfile

```diff
@@ -25,8 +26,8 @@ RUN ARCH=$(uname -m) && \
       libxslt-devel \
       make \
       openssl-devel \
-      python38-devel \
-      python38-pip \
+      python3-devel \
+      python3-pip \
```

Comment:
Wondering if we should make this python39 so that it builds in the version we expect as opposed to the version we happen to get.

Reply:
```
No match for argument: python39-devel
Error: Unable to find a match: python39-devel
```